recipes/pycppad/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "pycppad" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://github.com/Simple-Robotics/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8e3da0798a69b275a1cd0574939553fc65b152333e18832a791cfad199da0378
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - vs2017_win-64  # [win64]
+    - vs2017_win-32  # [win32]
+    - cmake
```

All point to the same
```suggestion
    - {{ compiler('cxx') }}
    - cmake
```